### PR TITLE
PIC-37 Adding DynamoDB

### PR DESCRIPTION
**Pull request by markberridge**

### CODE REVIEW

File: helm_deploy/court-case-service/templates/_envs.tpl [hunk 1]
```diff
@@ -19,4 +19,28 @@ env:
...
+        name: court-case-dynamodb-output
+        key: access_key_id
+
+  - name: DYNAMO_SECRET_KEY
+    valueFrom:
+      secretKeyRef:
+        name: court-case-dynamodb-output
+        key: secret_access_key
+
+  - name:
```
Comment (SWhitlam) [inline helm_deploy/court-case-service/templates/_envs.tpl]: you're missing the Env Var name here

File: src/main/java/uk/gov/justice/probation/courtcaseservice/CreateTableCommandLineRunner.java [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+import org.springframework.stereotype.Component;
+import software.amazon.awssdk.services.dynamodb.DynamoDbClient;
+import software.amazon.awssdk.services.dynamodb.model.*;
+import uk.gov.justice.probation.courtcaseservice.config.DynamoDbConfiguration;
+
+@Component
+@Profile("localDynamo")
+public class CreateTableCommandLineRunner implements CommandLineRunner {
+
+    private final Logger log = LoggerFactory.getLogger(CreateTableCommandLineRunner.class);
```
Comment (SWhitlam) [inline src/main/java/uk/gov/justice/probation/courtcaseservice/CreateTableCommandLineRunner.java]: You could use the lombok annotation here, but not sure if this is just a test class with it being a CLI runner

File: src/main/java/uk/gov/justice/probation/courtcaseservice/config/CourtCaseServiceProperties.java [hunk 3]
```diff
@@ -1,6 +1,5 @@
-package uk.gov.justice.probation.courtlistservice.config;
+package uk.gov.justice.probation.courtcaseservice.config;
```
Comment (SWhitlam) [inline src/main/java/uk/gov/justice/probation/courtcaseservice/config/CourtCaseServiceProperties.java]: Empty class

File: helm_deploy/court-case-service/templates/_envs.tpl (hunk 1 above)
Comment (markberridge) [reply helm_deploy/court-case-service/templates/_envs.tpl]: We don't need the ARN so I have removed it

**Pull request merged**